### PR TITLE
feat:  build using rolldown

### PR DESCRIPTION
**Pull request by underfin**

### CODE REVIEW

File: packages/vite/package.json [hunk 1]
```diff
@@ -88,7 +88,7 @@
     "esbuild": "^0.21.3",
     "postcss": "^8.4.39",
     "rollup": "^4.13.0",
-    "rolldown": "^0.10.5"
+    "rolldown": "/Users/likui/Github/rolldown/packages/rolldown"
```
Comment (IWANABETHATGUY) [inline packages/vite/package.json:91]: You should not use the local absolute path here.

**Comment by IWANABETHATGUY** [comment]
We should disable all the `vite` CI by default, and enable them on demand.

File: packages/vite/package.json (hunk 1 above)
Comment (underfin) [reply packages/vite/package.json:91]: Yeah. I know it. It will be replaced for nightly release.

**Comment by IWANABETHATGUY** [comment]
Let's merge it first.

**Pull request merged**